### DISCUSSION TITLE
airbyte data loading only first time from mongo to s3

### DISCUSSION THREAD

**Discussion by mohithsairamreddy** [airbytehq/airbyte#39543] [Connector Questions]
We were only able to sync data from mongodb to s3 for first time we are unable to sync data that was added consequently for option Append Historical Changes suboption Append New Rows and Updates Only . We were unable to capture records that were newly updated frommongodb

source : MongoDB Atlas
destination : S3 parquet

**Comment by riki-rsp** [comment]
Same here.

[a comment by an automated account is not shown]
